filters/color-gradient/src/ColorGradientFilter.ts

```diff
@@ -54,6 +54,14 @@ class ColorGradientFilter extends Filter
         maxColors: 0,
     };
 
+    public static fromCssGradient(cssGradient: string, options?: Partial<ColorGradientFilterOptions>) : ColorGradientFilter
```

Comment:
The static from method feels a little clunky and anti-pattern to me. No other filters do this. My suggestion is to have different option shapes a user could pass.

```ts
type ColorGradientFilterOptions = {
    type: number;
    stops: ColorStop[];
    angle: number;
    alpha: number;
    maxColors: number;
} | {
    css: string;
    alpha: number;
    maxColors: number;
};
```
Then in the constructor you could parse the CSS like you're doing here.